wfe/web-front-end.go

```diff
@@ -170,14 +171,14 @@ func (wfe *WebFrontEndImpl) HandleFunc(mux *http.ServeMux, pattern string, h fun
 		}
 		response.Header().Set("Access-Control-Allow-Origin", "*")
 
-		switch request.Method {
-		case "HEAD":
+		// Return a bodyless response to HEAD for any resource that allows GET.
+		if _, ok := methodsOK["GET"]; ok && request.Method == "HEAD" {
```

Comment:
Can you give a specific path that erroneously sends a body in response to a HEAD when methodsOK does not include GET? I tried new-authz and new-reg, but they both seem to behave correctly.